libindy/include/indy_wallet.h

```diff
@@ -216,6 +216,70 @@ extern "C" {
                                           void          (*fn)(indy_handle_t xcommand_handle, indy_error_t err, const char *const wallets)
                                           );
 
+    /// Exports opened wallet's content using key and path provided in export_config_json
+    ///
+    /// #Params
+    /// handle: wallet handle returned by indy_open_wallet.
+    /// export_config_json: JSON containing settings for input operation.
+    ///   {
+    ///     "path": path of the file that contains exported wallet content
+    ///     "key": passphrase used to export key
+    ///   }
+    ///
+    /// #Returns
+    /// Error code
+    ///
+    /// #Errors
+    /// Common*
+    /// Wallet*
+
+    extern indy_error_t indy_export_wallet(indy_handle_t  command_handle,
+                                           indy_handle_t  handle,
```

Comment:
> handle 

it is better to rename to wallet_handle

Reply:
i used existing conventions, same as in close_wallet